cpp/arcticdb/util/test/generators.hpp

```diff
@@ -54,7 +54,7 @@ struct SinkWrapperImpl {
             SchemaPolicy{
                 index_.create_stream_descriptor(std::move(stream_id), fields), index_
             },
-            [=](
+            [=, this](
```

Comment:
We only need sink, so we should either capture it minimally by value [sink=sink_] or just capture 'this'

Reply:
Shouldn't we capture `sink_` by reference?

Reply:
For now, e8cb5f7d532e77c1679f69e924881cf73fcec9c7 adapts the lambda only to capture `this`.